uniffi_bindgen/src/bindings/kotlin/templates/ObjectTemplate.kt

```diff
@@ -105,13 +105,21 @@
 {%- let (interface_name, impl_class_name) = obj|object_names(ci) %}
 {%- let methods = obj.methods() %}
 {%- let interface_docstring = obj.docstring() %}
+{%- let is_error = ci.is_name_used_as_error(name) %}
+{%- let ffi_converter_name = obj|ffi_converter_name %}
 
 {%- include "Interface.kt" %}
 
 {%- call kt::docstring(obj, 0) %}
+{%- let super_init = "" %}
+{% if (is_error) %}
+{%- let super_init = ": super()" %}
```

Comment:
I'm not sure this does anything because of scoping rules and a quick check in the generated kotlin seems to bear that out.  Maybe it's not needed or maybe the above `let` could change to be a ternary expression.

Reply:
thanks - apparently not needed - probably was in an earlier confused iteration!